test/cases/compile_errors/type_variables_must_be_constant.zig

```diff
@@ -7,5 +7,5 @@ export fn entry() foo {
 // backend=stage2
 // target=native
 //
-// :1:5: error: variable of type 'type' must be const or comptime
-// :1:5: note: types are not available at runtime
+// :1:11: error: variable of type 'type' must be const or comptime
+// :1:11: note: types are not available at runtime
```

Comment:
should this say "global variable"?
was the column change intentional?

Reply:
This wording is fine, and again, certainly need not be changed by this PR.

The column change was intentional, which is why it is reflected in the test case.